src/components/inputs/localized-multiline-text-input/localized-multiline-text-input.js

```diff
@@ -95,9 +94,12 @@ export class LocalizedMultilineTextInput extends React.Component {
     // This state is used to show/hide the remaining translations
     areLanguagesOpened: this.props.defaultExpandLanguages,
     // This state is to manage the expand/collapse of multiline text inputs
-    expandedTranslations: mapValues(this.props.value, () =>
-      Boolean(this.props.defaultExpandMultilineText)
-    ),
+    expandedTranslations: Object.keys(this.props.value).reduce((acc, key) => {
+      return {
+        [key]: Boolean(this.props.defaultExpandMultilineText),
+        ...acc,
```

Comment:
Can we maybe be more precise on naming the variables of the `reduce` ?

`acc` -> translations
`key` -> translation

🤔 

Also, this took me by surprised because I'm used to have code first the `...acc` and then the assignment 😅

Reply:
It's not translations though... The object looks like this

```
en: true,
es: true,
```
It's key value of locale and whether or not it's expanded...

Reply:
Then  `allLocales` maybe?